packages/app/package.json

```diff
@@ -82,8 +82,8 @@
     "@roadiehq/backstage-plugin-travis-ci": "^2.0.5",
     "history": "^5.0.0",
     "prop-types": "^15.7.2",
-    "react": "^17.0.2",
-    "react-dom": "^17.0.2",
+    "react": "^17.0.0 || ^18.2.0",
+    "react-dom": "^17.0.0 || ^18.2.0",
```

Comment:
It's up to the app to decide which React version to use, so here we'd use an exact version rather than a range

Reply:
Okay, I'll adjust that.